src/components/Shifting/ShiftingList.tsx

```diff
@@ -49,51 +53,172 @@ export default function ListView() {
   } = useQuery(routes.listShiftRequests, {
     query: formatFilter({
       ...qParams,
-      limit: resultsPerPage,
       offset: (qParams.page ? qParams.page - 1 : 0) * resultsPerPage,
     }),
   });
 
-  const showShiftingCardList = (data: ShiftingModel[]) => {
-    if (loading) {
-      return <Loading />;
-    }
-
-    if (!data || data.length === 0) {
+  const showShiftingCardList = (data: any) => {
```

Comment:
didn't catch this earlier... let's never downgrade from being type-safe to not being type-safe. using `any` type is not allowed (especially when it previously had a proper type)

Reply:
```const showShiftingCardList = (data: ShiftingModel[]) => {
    if (loading) {
      return <Loading />;
    }
    if (data && !data.length) {
      return (
        <div className="mt-64 flex flex-1 justify-center text-secondary-600">
          {t("no_patients_to_show")}
        </div>
      );
    }

    return data.map((shift: ShiftingModel) => (```